src/main/java/us/tastybento/bskyblock/listeners/PanelListenerManager.java

```diff
@@ -28,11 +28,11 @@ public void onInventoryClick(InventoryClickEvent event) {
         //UUID playerUUID = player.getUniqueId();
         Inventory inventory = event.getInventory(); // The inventory that was
         // Open the inventory panel that this player has open (they can only ever have one)
-        if (openPanels.containsKey(user.getUniqueId())) {
+        if (getOpenPanels().containsKey(user.getUniqueId())) {
```

Comment:
That's good to make openPanels private, though you can still use openPanels instead of the method within the PanelListenerManager

Reply:
Yes, it's just a style thing. The Eclipse IDE does it by default.